src/app/shared/data/projects.service.ts

```diff
@@ -33,6 +33,12 @@ export class ProjectsService {
     return this.projectsSubject.getValue();
   }
 
+  getProjectNameById(id: string): string {
+    const projects = this.getProjects();
+    const project = projects.find(project => project.id === id);
+    return project.name;
+  }
+
   // TODO: Delete after removing person and team code
   getProjectIdByName(name: string): string {
     const projects = this.projectsSubject.getValue();
```

Comment:
Subjects, Observables and so on should be postfixed with a dollar sign to let you know it is something you can subscribe to. --> `projectSubject$`